force-app/main/default/classes/CourseRegistrationHandler.cls

```diff
@@ -0,0 +1,135 @@
+global class CourseRegistrationHandler extends MyTriggers {
```

Comment:
Har du glemt å installere npm? Koden er ikke formatert. Se på #crm-dev for hvordan man gjør det.